articles/cms/wordpress/extending.md

```diff
@@ -209,17 +248,124 @@ CSS is retrieved from the wp-admin settings page.
 
 ```php
 /**
- * @param string $css, initialized as empty
+ * Add CSS to the Auth0 login form.
+ *
+ * @param string $css - initialized as empty.
  *
  * @return string
  */
-function a0_docs_ex_auth0_login_css( $css ) {
-	// ... modify $css by replacing or concatenating
+function auth0_theme_hook_auth0_login_css( $css ) {
+	$css .= '
+		body {background: radial-gradient(#01B48F, #16214D)} 
+		#login h1 {display: none}
+		.login form.auth0-lock-widget {box-shadow: none}
+	';
 	return $css;
 }
-add_filter( 'auth0_login_css', 'a0_docs_ex_auth0_login_css' );
+add_filter( 'auth0_login_css', 'auth0_theme_hook_auth0_login_css' );
+```
+
+### auth0_login_form_tpl
+
+Filters the template used for the Auth0 login form. This should return a path to a file containing HTML that replaces what is in `wp-content/plugins/auth0/templates/auth0-login-form.php`. The standard Lock initiation JS looks for an ID attribute of `auth0-login-form` to instantiate the login form so make sure that's present or replace the `wp-content/plugins/auth0/assets/js/lock-init.js` file with your own. 
+
+```
```

Comment:
set the scripting lang to ensure proper formatting

Reply:
@mpaktiti - I'm not sure how to do that ... also, this is just generic, not actual code. Should I use a different format?

Reply:
I'll set it to `php` like the rest of the examples in this page. Doesn't really make a big difference, it's just that if they don't have any sometimes they look weird.